cdapython.py

```diff
@@ -161,14 +161,14 @@ def columns(version=table_version, host=CDA_API_URL):
         return [list(t.values())[0] for t in query_result]
 
 
-def unique_terms(col_name, system=None, version=table_version, host=CDA_API_URL):
+def unique_terms(col_name, system=''):
     with cda_client.ApiClient(
-            configuration=cda_client.Configuration(host=host)
+            configuration=cda_client.Configuration(host=CDA_API_URL)
     ) as api_client:
         api_instance = QueryApi(api_client)
-
+        api_response = api_instance.unique_values(version=table_version, body=col_name, system=system)
```

Comment:
Why not cast the system param to a string you can use str() to make sure it is a str so i will not fail in a post to the server?

Reply:
also, pass it directly to the unique_values method as str(system), no need to assign variable if it is referenced only once below